### PR TITLE
Live tests/v5

### PR DESCRIPTION
**Pull request by victorjulien**
#11073, plus a libpcap live job based on the af-packet script.

### CODE REVIEW

**Comment by catenacyber** [comment]
Should the old PRs get closed ?

File: .github/workflows/live/pcap.sh [hunk 1]
```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# Script to test live IDS capabilities for OCAO. Starts a ping, starts suricata,
```
Comment (jasonish) [inline .github/workflows/live/pcap.sh:3]: Whats OCAO?

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 20620

File: .github/workflows/live/pcap.sh (hunk 1 above)
Comment (victorjulien) [reply .github/workflows/live/pcap.sh:3]: typo, meant to say "PCAP"

**Comment by victorjulien** [comment]
replaced by #11075

**Pull request closed without merging**